extension/page-detect.js

```diff
@@ -1,7 +1,7 @@
 /* globals utils */
 
 window.pageDetect = (() => {
-	const isGist = () => location.hostname === 'gist.github.com';
+	const isGist = () => location.hostname.startsWith('gist.') || location.pathname.startsWith('gist/');
```

Comment:
This will match any page on github.com for an organisation or person whose username start with gist. We should only check this when the domain is not github.com.